skimage/filters/thresholding.py

```diff
@@ -366,3 +368,89 @@ def threshold_li(image):
             new_thresh = temp + tolerance
 
     return threshold + immin
+
+def threshold_minimum(image, nbins=256, bias='min'):
+    """Return threshold value based on minimum method.  A histogram is computed
+    and smoothed until there are only two maximums.  Then the minimum between
+    these is found.
+
+    Parameters
+    ----------
+    image : array
+        Input image.
+    nbins : int, optional
+        Number of bins used to calculate histogram. This value is ignored for
+        integer arrays.
+    bias : {'min, 'mid', 'max}, optional
+        'min', 'mid', 'max' return lowest, middle, or highest pixel value
+        with minimum histogram value
+
+    Returns
+    -------
+    threshold : float
+        Computed threshold value.
+        May return 0 if the algorithm fails.
```

Comment:
Argh!  I have mostly been working with unsigned char data.  If there is an image with positive and negative values then zero can be a valid output.  Would you suggest throwing an exception instead, or returning a {success, threshold} tuple?